pkg/kf/internal/tools/clientgen/gentest/zz_generated.client.go

```diff
@@ -49,10 +50,8 @@ const (
 	APIVersion = "v1"
 )
 
-var (
-	ConditionReady       = apis.ConditionType("Ready")
-	ConditionInitialized = apis.ConditionType(v1.PodInitialized)
-)
+var ConditionReady = apis.ConditionType("Ready")
+var ConditionInitialized = apis.ConditionType(v1.PodInitialized)
```

Comment:
Is this the preferred way of declaring globals? Or was this just a formatting side-effect?

Reply:
This was me trying to avoid an empty `var ()` which was causing our code formatters to freak out, I found an alternate way with templates. I forgot that they evaluate empty objects as falsey.